sw/host/tests/chip/spi_device/BUILD

```diff
@@ -0,0 +1,56 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@rules_rust//rust:defs.bzl", "rust_binary")
+
+package(default_visibility = ["//visibility:public"])
+
+rust_binary(
+    name = "spi_device_flash_smoketest",
+    srcs = [
+        "src/spi_device_flash_smoketest.rs",
```

Comment:
Should the named binaries like this one be located at `src/bin/*.rs`? It looks like that would follow the cargo's conventions for package layout (and maybe satisfy clippy, though I'm not sure if we need a default executable too...).